spec/unit/rest/polling_spec.rb

```diff
@@ -75,7 +75,7 @@
 
   it 'should make 1 additional request when hitting 500' do
     expect do
-      @client.get('/internal_error')
+      @client.get('/internal_error', :tries => 3)
```

Comment:
because of https://github.com/gooddata/gooddata-ruby/pull/701/files#diff-b3fa27bc6eddfdd4e184fd694324e069R80